client/src/common/utils/localStorage.js

```diff
@@ -1,18 +1,42 @@
-export const loadSettings = () => {
+import { enumerable } from 'common/utils/helpers';
+
+export const storageKeys = enumerable()('ACCOUNT', 'SETTINGS');
+
+export const loadSettingsData = () => {
+  try {
+    const serializedState = localStorage.getItem(storageKeys.SETTINGS);
+
+    return serializedState === null ? undefined : JSON.parse(serializedState);
+  } catch {
+    // Ignore write errors
+  }
+};
+
+export const saveSettingsData = value => {
+  try {
+    const serializedState = JSON.stringify(value);
+
+    localStorage.setItem(storageKeys.SETTINGS, serializedState);
+  } catch {
+    // Ignore write errors
+  }
+};
+
+export const loadAccountData = () => {
   try {
-    const serializedState = localStorage.getItem('settings');
+    const serializedState = localStorage.getItem(storageKeys.ACCOUNT);
 
     return serializedState === null ? undefined : JSON.parse(serializedState);
   } catch {
     // Ignore write errors
```

Comment:
Please keep the single logic for load/save and the just:
```javascript
export const loadAccountData = load(storageKeys.ACCOUNT);
export const saveAccountData = save(storageKeys.ACCOUNT);
export const loadSettingsData = load(storageKeys.SETTINGS);
export const saveSettingsData = save(storageKeys.SETTINGS);
```

where load and save are generalized:

```javascript
export const load = storageKey => () => { ... }
export const save = storageKey => value => { ... }
```